src/common/Pagination/index.js

```diff
@@ -0,0 +1,47 @@
+import {
+  ArrowLeft,
+  ArrowRight,
+  Button,
+  ButtonText,
+  PageCounter,
+  PageNumber,
+  Wrapper,
+} from "./styled";
+
+const Pagination = ({
+  currentPage,
+  totalPages,
+  onFirstPage,
+  onPrevPage,
+  onNextPage,
+  onLastPage,
+}) => {
+  return (
+    <Wrapper>
+      <Button onClick={onFirstPage} disabled={currentPage === 1}>
+        <ArrowLeft />
+        <ButtonText>First</ButtonText>
+      </Button>
+      <Button onClick={onPrevPage} disabled={currentPage === 1}>
+        <ArrowLeft />
+        <ButtonText>Previous</ButtonText>
+      </Button>
+      <Button>
+        <PageCounter>
+          Page <PageNumber>{currentPage}</PageNumber>
+          of <PageNumber>{totalPages}</PageNumber>
+        </PageCounter>
+      </Button>
```

Comment:
I don't think it's supposed to be inside of a Button component.

Reply:
I removed the button. I guess I got carried away :-)